core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2590,7 +2590,7 @@ object SparkContext extends Logging {
   }
 
   /** Return the current active [[SparkContext]] if any. */
-  private[spark] def getActive: Option[SparkContext] = {
+  def getActive: Option[SparkContext] = {
```

Comment:
Hi, @nishkamravi2 .
Like the other review comments, I'm also wondering why you need this.
I'd like to use the existing official API instead of making another public API.
```scala
scala> org.apache.spark.sql.SparkSession.getActiveSession.get.sparkContext
res7: org.apache.spark.SparkContext = org.apache.spark.SparkContext@298154d4
```

We recommend users to use `SparkSession` instead of `SparkContext` if possible. And, we provide the API you need like the above.